cli/lib/info.js

```diff
@@ -5,8 +5,9 @@
  * See the LICENSE file for more information.
  */
 
-const
-	appc = require('node-appc'),
```

Comment:
Sorry, this is my super old coding style. It's bad. Nowadays I specify `const` or `let` per variable.